terraform/sample_image/test.py

```diff
@@ -0,0 +1,43 @@
+import json
+from os import environ as ENV
+from datetime import datetime
```

Comment:
Very minor: imports should be divided up by third party, standard library, and local scripts.